packages/messages-react/documentation/Messages.mdx

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
Not sure how the props `maxWidth` and `paddingLeft` should be exposed and documented from `SystemMessage`